describe.spec.ts

```diff
@@ -22,6 +27,7 @@ const privateElement = testScl.querySelector("Private")!;
 const sclElement = testScl.querySelector("SCL")!;
 const SomeNonSCLElement = testScl.querySelector("SomeNonSCLElement")!;
 const textElement = testScl.querySelector("Text")!;
+const enumValElement = testScl.querySelector("EnumVal")!;
 
 describe("Describe SCL elements function", () => {
   it("returns undefined with missing describe function", () =>
```

Comment:
In terms of test names, I'd rather have them read like

```
describe(element) -> is undefined for non-namespaced non-SCL elements
describe(element) -> describes EnumVal elements
```

etc. What do you think about such a naming scheme?

Reply:
To be honest with you, I do not understand your proposal, sorry. I am very open to discuss this further.